app/assets/javascripts/quote.js

```diff
@@ -39,13 +39,22 @@ class Quote {
     this.updateStaticMap(quoteData.polygons);
 
     document.getElementById("areaModal").innerText = `${quoteData.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
+    document.getElementById("subTotalModal").innerText = `${quoteData.subTotal.toLocaleString(undefined, {maximumFractionDigits: 2})}`;
+    document.getElementById("serviceExpeditionDetails").innerText = `${quoteData.serviceExpeditionDetails} ${quoteData.serviceExpeditionTime}`;
+    document.getElementById("serviceExpeditionCost").innerText = `${quoteData.serviceExpeditionCost}`;
+    document.getElementById("saltBagsQuantityModal").innerText = `${quoteData.saltBagsQuantity}`;
+    document.getElementById("saltBagsDueModal").innerText = `${quoteData.saltBagsDue}`;
```

Comment:
Why are all of these using interpolation (i.e. `` `${...}` ``? Can't you just set the `innerText` equal to whatever it is you're passing in? e.g.
```javascript
document.getElementById("subTotalModal").innerText = quoteData.subTotal.toLocaleString(undefined, {maximumFractionDigits: 2});
    
```

Reply:
This happens if i don't interpolate
<img width="482" alt="screen shot 2019-01-16 at 7 55 25 pm" src="https://user-images.githubusercontent.com/43128942/51294654-c4f42900-19c8-11e9-88e0-911a6e160811.png">